scripts/etherlink-bench.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2024-2025 TriliTech <contact@trili.tech>
+# SPDX-FileCopyrightText: 2025 Nomadic Labs <contact@nomadic-labs.com>
+#
+# SPDX-License-Identifier: MIT
+
+set -e
+
+ITERATIONS="1"
```

Comment:
Does this ITERATIONS variable do anything?

Reply:
No, I removed it now.